packages/mapviewer/src/modules/map/components/cesium/CesiumInteractions.vue

```diff
@@ -233,6 +243,62 @@ function onClick(event) {
     viewer.scene.requestRender()
 }
 
+function create3dKmlFeature(viewer, kmlFeature, kmlLayer) {
```

Comment:
Might make sense to add a little JSDoc here and at least describe what types each inputs are (it's not TS yet, so we need some help to understand better what's happening 😉 )